src/convert.ls

```diff
@@ -220,7 +220,7 @@ convert-question = (question, context, prefix = []) ->
   # range parameters.
   if question.type is \range
     select-range = (delete question.selectRange)
-    question.parameters = { start: select-range?.min, end: select-range?.max, step: (delete question.selectStep) }
+    question.parameters = { start: +select-range?.min, end: +select-range?.max, step: +(delete question.selectStep) }
```

Comment:
The explicit cast to numeric is the main and only source code change of this PR.